Task 3/Return subsequences of a string.

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
```

Comment:
The path of this file needs to be `Task 3/Return Sub sequences/ReturnSubsequencesOfString.cpp`